service/history/workflow/delete_manager.go

```diff
@@ -268,7 +266,7 @@ func (m *DeleteManagerImpl) archiveWorkflowIfEnabled(
 
 	// TODO: @ycyang once archival backfill is in place cluster:paused && namespace:enabled should be a nop rather than a delete
 	if !archiveHistory {
-		return true, nil
+		return false, nil
```

Comment:
Add comment to this method to explain what the return value means.

Reply:
updated